examples/src/java/org/pantsbuild/example/README.md

```diff
@@ -624,12 +624,18 @@ a simple 'badness' score. The "badness" score is intended to indicate both how e
 would be to remove (based on the maximum fraction used by each dependee) and how valuable it would
 be remove (based on a estimate of the transitive cost to build the dep).
 
+Stat explanation:
+
+* `max_usage`: percentage of the dependency target is being used. 0 means the target is safe to drop.
```

Comment:
This is the max within the entire context.

Also, it's a fraction, not a percentage.

Reply:
corrected.